angr/analyses/decompiler/clinic.py

```diff
@@ -18,6 +18,10 @@
 class Clinic(Analysis):
     """
     A Clinic deals with AILments.
+
+    :param angr.knowledge.Function func:
+    :param list<OptimizationPass> optimization_passes:
```

Comment:
There is real syntax you can use here! https://www.python.org/dev/peps/pep-0484/

`List[OptimizationPass]`

Reply:
+ many similar below